raster/r.grow.distance/r.grow.distance.html

```diff
@@ -64,6 +64,15 @@ <h2>NOTES</h2>
 to use it along with the <em>-m</em> flag in order to output 
 distances in meters instead of map units.
 
+<p>
+If <b>minimum_distance</b> is given, all cells with a distance smaller
+than <b>minimum_distance</b> will be set to NULL.
+
+<p>
+If <b>maximum_distance</b> is given, all cells with a distance larger
+than <b>maximum_distance</b> will be set to NULL. The resultant output
+is equivalent to a buffer.
+
```

Comment:
Add an explanation what will be the outcome if both – min and max distance are given. Will it be a ring (doughnut)?

Reply:
I have updated the manual to explain the result if both min and max distance restrictions are given.